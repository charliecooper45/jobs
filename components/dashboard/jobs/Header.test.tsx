import { render } from "@testing-library/react";
import Home from "./Header";

describe("Header", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Home />);

    expect(asFragment()).toMatchSnapshot();
  });
});
