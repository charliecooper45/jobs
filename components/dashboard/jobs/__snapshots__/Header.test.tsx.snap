// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-7lqv2e"
  >
    <div
      class="css-1i59fc"
    >
      Title
    </div>
    <div
      class="css-11koujc"
    >
      Salary
    </div>
    <div
      class="css-11koujc"
    >
      Status
    </div>
    <div
      class="css-11koujc"
    >
      Last Updated
    </div>
    <div
      class="css-pav208"
    />
  </div>
</DocumentFragment>
`;
